src/components/ListItem/ListItem.js

```diff
@@ -1,24 +1,38 @@
 import { useState } from 'react'
 import styled from 'styled-components/macro'
 
+import ChallengeButton from '../ChallengeButton/ChallengeButton'
+
 const ListItem = ({ name, img, age, wins, losses }) => {
   const [showDetails, setShowDetails] = useState(false)
+  const [isChallenged, setIsChallenged] = useState(false)
 
-  const toggleDetails = () => {
-    setShowDetails(!showDetails)
+  const setShowDetailsTrue = () => {
+    setShowDetails(true)
+  }
+  const setShowDetailsFalse = () => {
+    setShowDetails(false)
+  }
+  const toggleChallenge = () => {
+    setIsChallenged(!isChallenged)
   }
-
   return (
-    <StyledListItem onClick={toggleDetails}>
-      {name}
+    <StyledListItem>
+      <ClickArea onClick={setShowDetailsTrue}>{name}</ClickArea>
```

Comment:
for readability purposes only you might want to consider renaming `ClickArea` into something more clear, e.g. `DetailButton` or something similar

Also, as of now, your clickable area is only 20px in height and thus not very 'thumb-friendly'. You may want to consider making the whole element clickable instead and add prevent bubbling on the inner buttons.
![Bildschirmfoto 2021-10-14 um 11 08 38](https://user-images.githubusercontent.com/89029056/137287017-08371cc3-5cdb-4df3-aeb5-adeb4b837956.png)